var/spack/repos/builtin/packages/coin3d/package.py

```diff
@@ -43,6 +50,14 @@ class Coin3d(AutotoolsPackage):
     variant("debug", default=False, description="Make debug build")
     variant("symbols", default=False, description="Enable debug symbols")
 
+
+class CMakeBuilder(spack.build_systems.cmake.CMakeBuilder):
+    def cmake_args(self):
+        args = []
```

Comment:
I'm guessing we'll need to add flags here...

Reply:
done